CHANGELOG.md

```diff
@@ -1,3 +1,304 @@
+
+  In such a scenario, Lokomotive reboots the worker nodes and applies the user data changes. To bring about
+  such a change:
+
+  1. Make user data changes(if any).
+  2. Set `ignore_worker_changes = false`.
+
+* User data changes and reprovisioning of worker nodes:
+
+  In such a scenario, Lokomotive forces reinstallation of worker nodes via PXE and applies the user data changes.
+  This requires a meaningful `pxe_commands` value configured for automation. To bring about such a change:
+
+  1. Make user data changes (if any).
+  2. Remove the file with worker node MAC address from cluster assets directory.
+  3. Set `ignore_worker_changes = false` in cluster configuration.
+  4. Set `pxe_commands` to appropriate value.
+
+  **NOTE**: Reprovisioning will reinstallation the operating system. If you have any stateful workloads running, this
+  step would result is data loss.
+
+### Updating from v0.8.0
+
+#### Cluster update steps
+
+> **NOTE:** Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a version older than `v0.8.0`, update to `v0.8.0` first and only then proceed with the update to `v0.9.0`.
+
+Execute the following steps in your cluster configuration directory:
+
+1. Download and install the lokoctl binary by following the
+  [v0.9.0 installation guide](https://github.com/kinvolk/lokomotive/blob/v0.9.0/docs/installer/lokoctl.md)
+  and verify the version using `lokoctl version`:
+
+  ```bash
+  v0.9.0
+  ```
+
+##### Update steps for Equinix Metal (formerly Packet)
+
+   a. Backup the Terraform state file:
+   ```bash
+   cd $assets_dir/terraform
+
+   # If using local backend
+   cp terraform.tfstate backup.tfstate
+
+   # If using AWS S3 backend
+   terraform state pull > backup.state
+   ```
+
+   b. Update Terraform provider from `packethost/packet` to `equinix/metal`:
+   ```bash
+   terraform state replace-provider packethost/packet equinix/metal
+   ```
+
+   c. Pull the latest state file (required, only if using S3 backend):
+   ```bash
+   terraform state pull > terraform.tfstate
+   ```
+
+   d. Replace all references of `packet_` with `metal_` in the state file:
+   ```bash
+    sed -i 's/packet_/metal_/g' terraform.tfstate
+   ```
+
+   e. Change the module name from `module.packet` to `module.equinixmetal` in
+   the state file:
+   ```bash
+   sed -i 's/module.packet/module.equinixmetal/g' terraform.tfstate
+   ```
+
+   f. Replace `packet` with `equinixmetal` in the cluster configuration file:
+   ```hcl
+   # old
+   cluster "packet" {
+   ...
+   }
+
+   # new
+   cluster "equinixmetal" {
+   ...
+   }
+   ```
+
+   g. Uninstall Packet CCM as we are replacing it with Cloud Provider Equinix Metal.
+   ```bash
+   helm uninstall packet-ccm --namespace kube-system
+   ```
+
+   g. Upgrade to Lokomotive v0.9.0.
+   ```bash
+   lokoctl cluster apply --skip-components --skip-pre-update-health-check
```

Comment:
And the note on the `--upgrade-kubelets=false`?

Reply:
I dont understand. Do we want to go ahead with the update with `--upgrade-kubelets=false`

Reply:
Since this was the default behavior before and now we have added this backwards incompatible change we need to let the users know what the new "default" behavior looks like.